crates/turbopack-ecmascript/src/tree_shake/graph.rs

```diff
@@ -447,7 +457,8 @@ impl DepGraph {
 
                     global_done.extend(cycle.iter().copied());
 
-                    groups.push((ids, Default::default()));
+                    let len = ids.len();
+                    groups.push((ids, Default::default(), len));
```

Comment:
Nit: for consistency?

```suggestion
                    groups.push((ids, FxHashSet::default(), len));
````

Or could we just leave this off and create an empty `FxHashSet` inside the `add_to_group` loop? It looks like this is always initialized to an empty set, and never read after that loop?